yb-voyager/cmd/exportData.go

```diff
@@ -805,6 +805,13 @@ func checkDataDirs() {
 		if err != nil {
 			utils.ErrExit("Failed to truncate tables in metadb: %s", err)
 		}
+		if changeStreamingIsEnabled(exportType) {
```

Comment:
shall we do the same: `ClearMigrationState()` for targetDB also in case of start-clean?

Reply:
we already do this in importData in `cleanImportState()` on startClean right? are you talking about anything else?